packages/cli/test/integration-3.test.ts

```diff
@@ -330,7 +330,7 @@ test('output logs with "short" output', async () => {
 
   // "short" format includes timestamps
   expect(
-    stdout.match(
+    stderr.match(
```

Comment:
**⚠️ Important note** 

This is because I've reused `client.output.print` in a shared utility function that is used by `vc deploy`, `vc inspect` and `vc logs`.

`vc logs` used to print directly to stdout.
Is that an issue @EndangeredMassa ?

cc @lucleray (we changed it this morning)

Reply:
I think we should keep using `stdout`. The idea is that `stdout` only contains the "output", so the logs in this case, while `stderr` contains the messaging.

So, I think we need to revert the change and keep using `console.log` for now 🤔